### ISSUE TITLE
about test data split

### ISSUE THREAD

**Issue by yuanhu246** [LirongWu/MAPE-PPI#5]
Dear author, you mention in your paper dividing the test data into three subsets based on whether two proteins have been present in the training data, including: (1) BS: both have been present; (2) ES: one of the proteins is present; (3) NS: neither occurs. How is this implemented and where is the code for this part?

**Comment by LirongWu** [comment]
This is not hard to implement, a simple indicator function can divide the test set into different subsets and then just calculate the metrics for each subset separately. This part of the code was not included in the Github submission. An example is as follows:

```
prot_seen = set(list(ppi_list[ppi_split_dict['train_index']].reshape(-1)))

BS_list = []
ES_list = []
NS_list = []

if ppi_list[index][0] in prot_seen and ppi_list[index][1] in prot_seen:
     BS_list.append(num)
elif ppi_list[index][0] not in prot_seen and ppi_list[index][1] not in prot_seen:
     NS_list.append(num)
else:
     ES_list.append(num)
```
